hw/dv/tools/dvsim/vcs.hjson

```diff
@@ -247,6 +247,10 @@
   build_fail_patterns: ["^Error-.*$"]
   run_fail_patterns:   ["^Error-.*$"] // Null pointer error
 
+  // The first item is the regular expression to research for the CPU time.
+  // The second item is the index to find the CPU time after grouping the matched pattern.
+  cpu_time_pattern: ["^CPU Time:.*(\\d+\\.\\d+).*;", 1]
```

Comment:
We could simply hardcode the pattern in tool-specific methods in `sim_utils.py`. WDYT?

Reply:
Updated! Yeah that sounds good.